src/Sulu/Bundle/RouteBundle/Controller/RouteController.php

```diff
@@ -108,7 +108,7 @@ public function cgetAction(Request $request)
         $id = $this->getRequestParameter($request, 'id', true);
 
         // optional parameter
-        $history = $this->getBooleanRequestParameter($request, 'history', false, false);
+        $history = $this->getBooleanRequestParameter($request, 'history', false) ?? false;
```

Comment:
```suggestion
        $history = $this->getBooleanRequestParameter($request, 'history', false);
```

Should not be required.